lib/src/main/java/scal/io/liger/view/ReviewCardView.java

```diff
@@ -112,7 +112,11 @@ public void onClick(View v) {
             @Override
             public void onClick(View v) {
                 if (mMediaCards.size() > 0) {
-                    if (mCardsPlayer != null && mCardsPlayer.isPlaying()) mCardsPlayer.stopPlayback();
+                    if (mCardsPlayer != null) {
```

Comment:
we should probably make this part of our style guide, it relaly does help when reading crash reports